src/wandbot/ingestion/prepare_data.py

```diff
@@ -790,19 +774,16 @@ def lazy_load(self) -> Iterator[Document]:
         Yields:
             A Document object.
         """
-        lang_detect = FastTextLangDetect()
         data_dump_fame = self.fetch_data()
         for parsed_row in self.parse_data_dump(data_dump_fame):
             document = Document(
                 page_content=parsed_row["content"],
                 metadata={
                     "source": parsed_row["source"],
-                    "language": lang_detect.detect_language(
-                        parsed_row["content"]
-                    ),
+                    "source_type": self.config.source_type,
                     "file_type": ".md",
                     "description": parsed_row["description"],
-                    "tags": ["fc-report"]
+                    "tags": ["Fully Connected", "Report"]
```

Comment:
same lowercased, hyphenated comment as above. also lets refer to 'reports' as articles from now

Reply:
same as above. It's easier to run contains query when using metadata filters with un-hyphenated strings.